consensus/src/chained_bft/event_processor_test.rs

```diff
@@ -225,7 +229,11 @@ fn basic_new_rank_event_test() {
     let runtime = consensus_runtime();
     let mut playground = NetworkPlayground::new(runtime.handle().clone());
     let mut nodes = NodeSetup::create_nodes(&mut playground, runtime.handle().clone(), 2);
-    let node = &mut nodes[0];
+    // This is ugly, but can't think of a better way of getting reference on
+    // two elements of same vector
+    let (node_slice, node_slice_1) = nodes.split_at_mut(1);
```

Comment:
we probably don't need this, just use nodes[0] and nodes[1] directly below probably works, or just pop them instead of split